cms-adapter/src/Rsbc.Dmf.CaseManagement/Dynamics/DynamicsContext.cs

```diff
@@ -18,7 +18,9 @@ public DynamicsContext(Uri serviceRoot, Uri url, Func<Task<string>> tokenFactory
             this.SaveChangesDefaultOptions = SaveChangesOptions.BatchWithSingleChangeset;  // Set to SaveChangesOptions.None to troubleshoot query issues
             MergeOption = MergeOption.OverwriteChanges;
 
-            this.EntityParameterSendOption = EntityParameterSendOption.SendOnlySetProperties;
+           // this.EntityParameterSendOption = EntityParameterSendOption.SendOnlySetProperties;
+
+             this.EntityParameterSendOption = EntityParameterSendOption.SendFullProperties;
```

Comment:
@GeorgeWalker will this have any unwanted side-effects with any other queries, since it's a global change?

Reply:
We are relying on GRPC to process the transactions sequentially here so it should be fine.

Reply:
Thanks George for the clarification. I have reverted the change in the dynamics Contextfile and added that in the code for unclaiming the DMER.